my-app/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import './App.css';
+
+class Board extends Component {
+  constructor() {
+    super();
+    this.size = 9;
+    this.state = {
+
+      // wIsNext: true,
```

Comment:
Consider omitting commented out code from review - such omissions decrease the visual noise in the commit, and make the code easier to scan.